### PR TITLE
Update Documentation for v0.8.0

### PR DESCRIPTION
**Pull request by vincentpun**
# Context

In v0.8.0 of the OctoML CLI, the local deployment feature has been removed, and configuration options such as the access token and telemetry options are now specified using environment variables. This pull request updates the README files to match the next version of the CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,29 @@
 # OctoML CLI
 
 The `octoml` CLI helps you create deployable containers for your ML models using the OctoML Platform, the terminal or within your CI/CD pipeline.
-You can run inferences on the container locally for development and testing, then deploy the same container to the cloud.
+You can run then deploy the same container to the cloud.
```
Comment (vanessahlyan) [inline README.md:4]: I think we can just remove run here and say "You can then deploy these containers directly to your cloud registry."

File: README.md [hunk 2]
```diff
@@ -40,37 +35,26 @@ octoml 0.7.6 (e13c6f5 2023-01-11 13:22:22)
 **Model framework coverage**: 
 We support TensorFlow SavedModel, TensorFlow GraphDef, Torchcript (PyTorch), and ONNX models.
 
-**Hardware coverage**: 
-You may run our CLI on x86 or CUDA machines. If you are using macOS, ARM64 machines are also supported (including M1 and M2 machines). In unauthenticated usage, the containers built via the CLI may only be deployed on hardware architectures matching the local machine on which you ran
```
Comment (vanessahlyan) [inline README.md:44]: why are we removing the whole section? we should keep the authenticated usage part, no?

Comment (vincentpun) [reply README.md:44]: Oh that's a good point. Updated.

File: README.md (hunk 1 above)
Comment (vincentpun) [reply README.md]: Updated 😃

**Review by vanessahlyan — APPROVED** [review]

**Pull request merged**